build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
       'assertj': '1.7.0',
       'shadowPlugin': '4.0.4',
       'nodePlugin': '1.2.0',
-      'ktlintPlugin': '5.1.0',
+      'ktlintPlugin': '7.1.0',
```

Comment:
Old version didn't like unsigned literals in the samples project. Not sure why it was okay before.